testapi.pm

```diff
@@ -1061,6 +1049,37 @@ sub script_output {
     return $distri->script_output($script, %args);
 }
 
+=head2 script_output_with_status
+
+  my ($out, $retval) = script_output_with_status($script [, timeout => ?] [, type_command => ?] [, proceed_on_failure => ?] [, quiet => ?])
+
+Executes a script inside SUT with C<bash -eox> (in case of serial console with C<bash -eo>)
+and directs C<stdout> (I<not> C<stderr>!) to the serial console. Returns an array
+with the data written to C<stdout> and the exit code of the script.
+I<If> the script exits with != 0, the test is set to failed. Use
+C<proceed_on_failure> to proceed with validation.
+If retrieved C<$out> is I<undef>, this function wasn't able to find the start
+and end marker to identify the script output.
+NOTE: execution result may include extra serial output which was on serial console
+since command was triggered in case serial console is not dedicated for
+the script output only.
+
+The given script is typed or fetched through HTTP depending on various parameters.
+Typing can be forced by passing C<<type_command => 1>> for example when the SUT does
+not provide a usable network connection.
+
+The default timeout for the script is based on the default in C<wait_serial>
+and can be tweaked by setting the C<$timeout> parameter.
+
+Use C<quiet> to avoid recording serial_results or test variable C<_QUIET_SCRIPT_CALLS>.
+=cut
+
+sub script_output_with_status {
+    my ($script, %args) = @_;
+    $args{quiet} //= testapi::get_var('_QUIET_SCRIPT_CALLS');
```

Comment:
Interesting => setting new standard to use underscore prefix for library variables. I like it, but it could be noted also in git commit message (I guess as it's in perldoc it will be in the docs).

Also adding new dependency (`Mojo::Util`) could be noted.

Reply:
This _QUIET_SCRIPT_CALLS comes from https://github.com/os-autoinst/os-autoinst/pull/1160
I thought `Mojo::Util` is a common mojo library and we are using it. So I thought it isn't a new one.